framework/web/ErrorAction.php

```diff
@@ -98,6 +105,8 @@ public function init()
      */
     public function run()
     {
+        if ($this->layout) $this->controller->layout = $this->layout;
```

Comment:
Code style must be PSR-2 compatible:

- [Yii 2 Core Framework Code Style](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md)
- [PSR-2 - if, elseif, else](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#51-if-elseif-else)